src/test/java/space/obminyashka/items_exchange/end2end/CategoryFlowTest.java

```diff
@@ -76,7 +76,7 @@ void getCategoryById_whenCategoryIdDoesNotExist_shouldReturnNotFound() throws Ex
 
     @Test
     @DataSet("database_init.yml")
-    void getCategorySizesById_whenCategoryIdDoesNotExist_shouldReturnBadRequest() throws Exception {
+    void getCategorySizesById_whenCategoryIdDoesNotExist_shouldReturnNotFound() throws Exception {
```

Comment:
Add @ExpectedDataSet to compare data

Reply:
Don't think that's possible. See: https://github.com/Jack11M/EVO-Exchange-BE-2019/pull/494#discussion_r1117786605